src/test/java/fi/aalto/cs/apluscourses/intellij/utils/CourseFileManagerTest.java

```diff
@@ -178,7 +178,7 @@ public void testLoad() throws IOException, InterruptedException {
     startAndJoinEach(threads);
 
     Assert.assertEquals("CourseFileManager returns the correct URL",
-        new URL("https://example.com"), manager.getCourseUrl());
+        new URL("https://example.org"), manager.getCourseUrl());
```

Comment:
but why :question:

Reply:
That's a really good question!

It failed for me, I looked at the line 155 and I reasoned the test was broken, so I "fixed" it. Now I see the test passes with either value! Which is insane. There's definitely something shady there that should be taken into scrutiny. Perhaps it could even make us understand the errors we had with `CourseFileManager` in production.